src/sentry/integrations/slack/tasks.py

```diff
@@ -118,6 +133,10 @@ def find_channel_id_for_rule(project, actions, uuid, rule_id=None, **kwargs):
             rule = project_rules.Updater.run(rule=rule, pending_save=False, **kwargs)
         else:
             rule = project_rules.Creator.run(pending_save=False, **kwargs)
+            if user:
```

Comment:
Just wondering if there should be a case we don't have a user and that maybe we should log an error (so we can fix it) or write a user-less activity if there isn't one